### ISSUE TITLE
Invalid manifest after removing a rock

### ISSUE THREAD

**Issue by rosik** [tarantool/rocks.tarantool.org#14]
See http://rocks.tarantool.org/manifest.
Now it contains the following record:

```lua
    cartridge = {
        ["0.0.0-1"] = {
            [2] = {
                arch = "all"
            }
        },
```

Table key `[2]` shouldn't be there, it's an array and it should start from 1.

**Comment by rosik** [comment]
I believe this bug should have been fixed in #8, but it's still there

**Comment by rosik** [comment]
It's ok after re-deploy. Checked on prod
